adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -73,6 +81,10 @@
     private static final int IANA_MQTT_PORT = 1883;
     private static final int IANA_SECURE_MQTT_PORT = 8883;
 
+    private static final String COMMAND_REQUEST_PREFIX = "control/req";
+    private static final String COMMAND_RESPONSE_PREFIX = "control/res";
+    private static final String COMMAND_REQUEST_SUBSCRIPTION = COMMAND_REQUEST_PREFIX + "/#";
```

Comment:
IMHO we should move these three constants to either `Constants` and make them public,
or at least make them protected, so that sub classes of this abstract class can use them.

Reply:
The question is if this is only MQTT or also other. Since I will need this in the ResourceIdentifier later, I agree to move it to Constants.